lib/wcgi-runner/examples/wcgi-runner.rs

```diff
@@ -0,0 +1,111 @@
+use std::{convert::Infallible, net::SocketAddr, path::PathBuf};
+
+use anyhow::{Context, Error};
+use clap::Parser;
+use tracing_subscriber::fmt::format::FmtSpan;
+use wcgi_runner::Runner;
+
+fn main() -> Result<(), Error> {
+    if std::env::var("RUST_LOG").is_err() {
+        std::env::set_var("RUST_LOG", "wcgi_runner=trace,info");
+    }
+    tracing_subscriber::fmt()
+        .with_span_events(FmtSpan::CLOSE)
+        .init();
+
+    let Args {
+        address,
+        input,
+        env_all,
+        mapped_dirs,
+    } = Args::parse();
+
+    // Hack to initialize the global shared tokio task manager handle.
+    // Prevents "cannot drop runtime in async context" errors, because
+    // the default will be initialized to the current tokio context.
+    let rt = wasmer_wasi::runtime::task_manager::tokio::TokioTaskManager::default();
+
+    let mut builder = Runner::builder()
+        .map_dirs(mapped_dirs)
```

Comment:
I'd put here: with_wasi_config(CONFIG). Because that way it can be reused in the future with the WASI runner